src/theme/assets/scss/theme/_base.scss

```diff
@@ -13,11 +13,15 @@ h1, h2, h3, h4, h5, h6 {
 	font-family: 'Poppins', sans-serif;
 	color: $theme-text-color-primary;
 	font-weight: 600;
+}
 
+/*! purgecss start ignore */
```

Comment:
See https://github.com/FullHuman/purgecss/blob/v2.3.0/docs/whitelisting.md

Tried without the `!` initially but it didn't work.